operator/redisfailover/service/client.go

```diff
@@ -61,8 +61,11 @@ func (r *RedisFailoverKubeClient) EnsureSentinelService(rf *redisfailoverv1alpha
 
 // EnsureSentinelConfigMap makes sure the sentinel configmap exists
 func (r *RedisFailoverKubeClient) EnsureSentinelConfigMap(rf *redisfailoverv1alpha2.RedisFailover, labels map[string]string, ownerRefs []metav1.OwnerReference) error {
-	cm := generateSentinelConfigMap(rf, labels, ownerRefs)
-	return r.K8SService.CreateOrUpdateConfigMap(rf.Namespace, cm)
+	if _, err := r.K8SService.GetConfigMap(rf.Namespace, GetSentinelName(rf)); err != nil {
```

Comment:
This should check first if the custom ConfigMap is set (hence the user wants to use it).

Reply:
if err not nil means custom ConfigMap is not exist and will use default ConfigMap.
Do I need to add other checks?